netlify-plugin-cloudinary/src/index.ts

```diff
@@ -134,9 +134,7 @@ export async function onBuild({
     cname,
     deliveryType,
     folder = process.env.SITE_NAME || '',
-    imagesPath = CLOUDINARY_ASSET_DIRECTORIES.find(
-      ({ inputKey }) => inputKey === 'imagesPath',
-    )?.path,
+    imagesPath = inputs.imagesPath || '/images',
```

Comment:
looking at this again - the intent for using `CLOUDINARY_ASSET_DIRECTORIES` was so that i could define the different configurations for different asset types in one location

for instance, in the future there could be a second object in `CLOUDINARY_ASSET_DIRECTORIES` with a name of `videos`

im kind of surprised the default wasn't working properly as the code seems to run fine

<img width="464" alt="image" src="https://github.com/cloudinary-community/netlify-plugin-cloudinary/assets/1045274/875ff21c-60bf-4131-ad8b-854d219a62ec">

can we keep the default value but use the `inputs` first if available like you have?

Reply:
> can we keep the default value but use the inputs first if available like you have?

I'll look into this now~

Reply:
✅